lib/view_component/render_preview_helper.rb

```diff
@@ -11,15 +11,25 @@ module RenderPreviewHelper
     # ```
     #
     # Note: `#rendered_preview` expects a preview to be defined with the same class
-    # name as the calling test, but with `Test` replaced with `Preview`:
+    # name as the calling test, but with "Test" replaced with "Preview":
```

Comment:
IMO we should use backticks for constants.

```suggestion
    # name as the calling test, but with `Test` replaced with `Preview`:
```